### ISSUE TITLE
Connection fail using rclone

### ISSUE THREAD

**Issue by fatshotty** [divyam234/teldrive#237]
Hi,
i'm using teldrive with rclone and everything works fine. Unfortunately, sometimes, teldrive gives errors:

`file streamwaitSession: connection dead`
and
`file streamrpcDoRequest: engine was closed`
in these cases, files are unreachable and rclone can't list files neither access to file
I need to stop and restart teldrive

is it only me?

**Comment by Linden10** [comment]
Hmm...I don't know if it's the same issue as I had in the past but in a now older version of teldrive, teldrive wouldn't work after putting the computer to sleep or such and files wouldn't stream. I had to simply restart teldrive itself to fix it.

Though in the newest versions of teldrive, this issue has been resolved and I never encountered the issue again.
I don't stream too much from teldrive though so I'm not sure if this is similar to what I had.

If you haven't, be sure to upgrade both teldrive and rclone if you haven't!
However if this is the newest version of teldrive you're encountering this issue with, it might be a problem...

In that case, share logs if upgrading to the newest version of teldrive/rclone hasn't fixed this issue!
